src/NativeMessageHost/CMakeLists.txt

```diff
@@ -92,6 +93,7 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
 
 if (FIREBREATH)
     link_boost_library(${PROJECT_NAME} filesystem)
+    link_boost_library(${PROJECT_NAME} regex)
```

Comment:
I'm not sure what is using regex there, but it would probably be better to change it to use std::regex (c++11 regex libs) rather than linking in another boost library.